### DISCUSSION TITLE
Can I set this up on my Mycroft Mark1? Please ELI5

### DISCUSSION THREAD

**Discussion by sanderant** [OpenVoiceOS/ovos-buildroot#135] [Q&A] [answered]
I've been using my Mark1 until everything unraveled recently and I'd like to use the same hardware, but trying to get a handle on what's involved since according to this response https://community.mycroft.ai/t/mark-i-options/13340/2 the Mark1 may not be supported yet I see https://github.com/OpenVoiceOS/ovos-PHAL-plugin-mk1 that there is some code.

I want to know will the Mark1 enclosure be supported? Can I just update the raspberry pi to a new release and install it like a regular client? Or do I just need to install the personal back end and use the existing set up? I'm hesitant to wipe the current set up since who knows what will be available in the future and at least it still works a clock.

TIA

**Comment by j1nx** [comment]
Hi there,

We are currently working on a headless image with the focus on the Raspberry Pi3. When the whole OS structure of the headless image is about done, we can look into the headless setup route.

With that in place, indeed we are looking into supporting the Mark1 with it. Much of the differen building block already included, they just need to be stitched together.

However with only 24 hours in a day, it is hard to give an ETA for it.

**Comment by sanderant** [comment]
Hey,

Thanks for the reply this clarifies things somewhat. In the long run I'd just like to use the chassis with some software. I was hoping I could just change some of the config or do a dist-upgrade or something, but it sounds like it will need more help than that.

I would only ask that you clarify the status of the Mark I from time to time and if you end up not being able to get to it just us know.

Thanks,

Sander

**Comment by JarbasAl** [comment]
I've had some success with very old pi3 builds of OVOS, but my mk1 faceplate is somewhat defective so testing has been hard and the mk1 plugin is pretty much untested, but it is a direct port of the old code from mycroft-core and should work. 

We are now starting to get ovos-buildroot pi3 builds and I will resume work on this. I very much want to support my Mark1 device and I have a bunch of updates for the faceplate as well, plenty animations and things like conways game of life implemented here https://github.com/OpenVoiceOS/ovos-utils/tree/dev/ovos_utils/enclosure/mark1

I will keep you updated in this thread as soon as there is any progress

**Comment by josefo2020** [comment]
Really happy you guys are working on supporting the Mark 1! It's such a unique and quirky device. 

Do you guys envision updating the mark1 to support the rpi4? It would be great to be able to update that hardware.

**Comment by JarbasAl** [comment]
as a side note, you can keep your stock mk1 running if mycroft goes down via personal backend https://openvoiceos.github.io/community-docs/personal_backend/

**Comment by ChanceNCounter** [comment]
Update: the OVOS team is attempting to acquire more Mark I units to help with this initiative. Still no ETA, but we have found some units, so there is progress.

**Comment by builderjer** [comment]
There is a [test image](https://ovosimages.ziggyai.online/raspbian/mark1/) available for the Mark 1.  I know there are a few issues, but don't remember exactly what they are.  I don't personally have one to test on, but would be happy to look at [issues made here](https://github.com/OpenVoiceOS/raspbian-ovos/issues) or even better [a PR](https://github.com/OpenVoiceOS/raspbian-ovos/pulls)

**Comment by builderjer** [comment, marked as answer]
@sanderant I am happy to say that OVOS now has native support for the Mark 1.  Check out the latest headless image from [here](https://ovosimages.ziggyai.online/raspbian/newest)

https://matrix.to/#/!EOygeDJPfJQOfNacqH:matrix.org/$YwJ8Kc-LaPvySL9efmJorIOuc0cv6lv0Pr_i5iEO6Xw?via=matrix.org&via=mozilla.org&via=sietch.online

**Comment by sanderant** [comment]
Thanks for following up on this. I have downloaded the image I will create a Matrix account and checkout the link.

**Comment by josefo2020** [comment]
Just got it working! Thanks guys.

The "hotspot" didn't work on startup to get it connected to wifi, but I used an ethernet connection to SSH in to set up the wifi.

**Reply by builderjer** [reply]
Strange the hotspot is not working for you.  Is your AP hidden? or are you on WiFi6?
